scripts/chart/build

```diff
@@ -14,3 +14,9 @@ cp -rf ${1} build/chart/rancher
 
 sed -i -e "s/%VERSION%/${CHART_VERSION}/g" build/chart/rancher/Chart.yaml
 sed -i -e "s/%APP_VERSION%/${VERSION}/g" build/chart/rancher/Chart.yaml
+
+post_delete_base=$(grep -i shell-image pkg/settings/setting.go | cut -d "," -f 2 | sed -e 's/"//g' | sed -e 's/)//g' | sed -e 's/ //g')
+post_delete_image_name=$(echo ${post_delete_base} | cut -d ":" -f 1)
+post_delete_image_tag=$(echo ${post_delete_base} | cut -d ":" -f 2)
```

Comment:
Let's add some checks here to ensure the name and tag are what we expect. We _could_ hardcode the image name but thats probably not necessary as it could change down the line. But checking that tag starts with v or that "shell" is in image name, something like that.

Reply:
The following 2 changes are made to address your comments:
1. For the line 18-20, those commands will not exit with code 0, they return an empty string and exit code 0 if fail.
2. The checks for the image name and tag are added at lines 21-28.